### ISSUE TITLE
question about 1 way training

### ISSUE THREAD

**Issue by JiXuKong** [fanq15/FSOD-code#13]
Thanks for your amazing work! Recently, **I have mimiced your 1 way 5 shot method in pascal voc dataset by tf1.12. But I finally find the calssifier hard to find the right quaried category.**  **I wonder if there's something wrong with my implement and my implement detail is show in below:**
       1. I use pretrained resnet50 as backboned, The first 3 blocks is used for feature extraction. After that, ROI pooling is used for support feature(using their groundtruth box), then avg support feature in their batch, height, width 3 dimensions, Finally do channel-wise attention, and the match relation is implemented as your code.
      2. **For every training image, I will randomly choose a category contained in it, then choose 5 spport cropped images(same class) which cropped from all training set, all the other calsses in the image treated as background.**
Thanks a lot!

**Comment by fanq15** [comment]
I think your mentioned implementation is right. There should be some bugs in your code. You should check your loss first to make sure the training loss drops correctly. You can check every step by printing out the result or plotting the image, e.g. the cropped support image, to check your code. You can also only use the basic component to simplify your procedure and make sure that your every step is right.

**Comment by JiXuKong** [comment]
Thanks, I find my loss hard to drop,thanks for your advice, I will check my code.
